SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -41,7 +41,7 @@ $DefaultNoteTypeMap = @{
 $lpassMessage = @{
     AccountNotFound = 'Error: Could not find specified account(s).'
     # Need to use wildcard since the path of lpass could be different
-    LoggedOut = 'Error: Could not find decryption key. Perhaps you need to login with*'
+    LoggedOut = 'Error: Could not find decryption key. Perhaps you need to login with Connect-LastPass'
```

Comment:
Why did you make this change? That star is needed to properly surface the right error.

Reply:
Indeed.
I'll revert it.
I missed the `-wildcard` and the fact it was not only the output message but also used in the switch statement. 

```suggestion
    LoggedOut = 'Error: Could not find decryption key. Perhaps you need to login with*'
```


That being said, I just found another thing. 
Since we do not redirect error stream anymore on the Get so we get the prompt, we won't get that message displayed ever again. 

(Because $result will always be $null so I am keeping this conversation open and I will review if we can indicate to the user it need to login instead of displaying: The secret <name> was not found **Figure-2)** 
![image](https://user-images.githubusercontent.com/1980296/99342250-78463700-2859-11eb-90f6-b54cd891a9bd.png)

**Figure-2**
![image](https://user-images.githubusercontent.com/1980296/99342416-cc511b80-2859-11eb-8126-6f6d56e0d0b8.png)

Reply:
@itfranck I guess I kept it in in the event that the user has _never_ logged in. Prompting for the password requires you to have logged in in the past AFAICT because otherwise it would need to prompt for username as well ... but maybe I have this wrong.

Reply:
I've created an issue and will deal with it specifically in its own PR so I can close this one. 

I think I have a good idea how to fix this. 
Will elaborate soon.